maintenance-filter-server/src/main/java/org/ccci/maintenance/util/MaintenanceWindowDao.java

```diff
@@ -0,0 +1,18 @@
+package org.ccci.maintenance.util;
+
+import org.ccci.maintenance.MaintenanceWindow;
+import org.joda.time.DateTime;
+
+/**
+ * @author Matt Drees
+ */
+public interface MaintenanceWindowDao
+{
+    MaintenanceWindow getActiveMaintenanceWindow(String filterName, DateTime currentDateTime);
+
+    /** retrieves the window identified by the given id, or null if it does not exist */
+    OwnedMaintenanceWindow getMaintenanceWindowById(String id);
```

Comment:
That makes sense for not wanting to use Guava for those reasons.  I've actually noticed instances where I've been burned on different versions of it, but not frequently enough that I realized they don't tend to care about backwards compatibility so much.

I am wholeheartedly against anything that remotely sounds like JarJar Binks in any of our projects. :)